tile.yml.erb

```diff
@@ -29,12 +31,11 @@ packages:
     instances: 1
     properties:
       rlp:
-        address: (( .properties.rlp_address.value ))
-        ca_cert: (( .properties.rlp_cacert.value ))
-        cert: (( .properties.rlp_cert.value ))
-        key: (( .properties.rlp_key.value ))
+        ca_cert: (( $ops_manager.ca_certificate ))
+        cert: (( .properties.rlp_creds.cert_pem ))
+        key: (( .properties.rlp_creds.private_key_pem ))
       firehose:
-        endpoint: (( .properties.firehose_endpoint.value ))
+        endpoint: (( $runtime.system_api_url ))
```

Comment:
I removed the firehose_endpoint property from the tile. I assumed that the API targets will be the same as the runtime deploying the tile. 

If this is something we should keep open for configuration, let me know.